scripts/local/launch/helpers/create-azure-resources.sh

```diff
@@ -73,7 +73,7 @@ function main {
 	set -e
 	check_dependencies
 	create_resources
-	copy_env
+	#copy_env
```

Comment:
Remove?

Reply:
This i am keeping if reqd in future. Currently plan is to save the env in a file.